status/views.py

```diff
@@ -0,0 +1,28 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from django.template import RequestContext
+
+from problem.models import Submission, SubmissionDetail
+from index.views import custom_proc
+# Create your views here.
+
+
+def status(request):
+    submission_details = SubmissionDetail.objects.all()
+    submission_details = submission_details.extra(order_by=['-sid'])
```

Comment:
These two lines (11&12) can be combined. Use django build-in function 'order_by'.
https://docs.djangoproject.com/en/1.7/ref/models/querysets/#django.db.models.query.QuerySet.order_by